source/supporting-ie8/index.html.md.erb

```diff
@@ -17,9 +17,7 @@ then you will need to do some additional work to support Internet Explorer 8
 Once you have completed these steps, you will be able to [write CSS that targets
 IE8 in your own application styles](#writing-styles-that-target-ie8).
 
-If you are using the distributed versions of GOV.UK Frontend that already
-include an IE8-specific stylesheet then you can include that in your project,
-allowing you to skip steps 2 and 3.
+If you [installed using precompiled files](/install-using-precompiled-files), you can include the IE8-specific stylesheet in your project, allowing you to skip steps 2 and 3.
```

Comment:
This is the only page that mentions [installing using precompiled files](/install-using-precompiled-files) AND links to it. Could we link to the guidance from the other pages too to help users understand what we mean? Seeing "installing using precompiled files" without a link might sound a bit confusing and make users question whether it's something they've done or not.

Reply:
I think the main places we mention precompiled files without a link is where we warn users from using them (for example in migrating or using compatibility mode). So it's probably best to avoid a link there, to avoid users potentially heading off and doing a thing they shouldn't.

I may have missed some mentions though - let me know if so!

Reply:
Okay happy to go with your hunch here 👍